flattentool/input.py

```diff
@@ -555,7 +560,10 @@ def get_sheet_lines(self, sheet_name):
 
 class XLSXInput(SpreadsheetInput):
     def read_sheets(self):
-        self.workbook = openpyxl.load_workbook(self.input_name, data_only=True)
+        try:
+            self.workbook = openpyxl.load_workbook(self.input_name, data_only=True)
+        except BadZipFile as e:
+            raise Exception("The supplied file has extension .xlsx but isn't an XLSX file.")
```

Comment:
Can we use a custom exception type here, to make it possible to specifically catch this in CoVE without needing to ammend flatten-tool further?